ios/browser/api/sync/prefs/BUILD.gn

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2021 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import("//build/config/ios/rules.gni")
+import("//ios/build/config.gni")
+
+source_set("prefs") {
+  configs += [ "//build/config/compiler:enable_arc" ]
+
+  sources = [
+    "brave_sync_profile_service.h",
+    "brave_sync_profile_service.mm",
+    "brave_sync_profile_service_private.h",
+  ]
+
+  deps = [
+    "//base",
+    "//components/sync/base",
+    "//components/sync/driver",
+    "//components/sync_device_info",
+    "//ios/chrome/browser",
+    "//ios/chrome/browser/browser_state",
+    "//ios/chrome/browser/sync",
+    "//ios/web/public/thread",
+  ]
+
+  frameworks = [
+    "Foundation.framework",
+    "UIKit.framework",
```

Comment:
do you need UIKit here? I only see Foundation.h

Reply:
Yes we dont need `UIKit framework` here, it is removed